yjit/src/codegen.rs

```diff
@@ -7891,7 +7928,7 @@ fn gen_getblockparam(
     asm.test(flags_opnd, VM_ENV_FLAG_WB_REQUIRED.into());
 
     // if (flags & VM_ENV_FLAG_WB_REQUIRED) != 0
-    asm.jnz(Target::side_exit(Counter::gbpp_wb_required));
+    asm.jnz(Target::side_exit(Counter::gbp_wb_required));
```

Comment:
> Revert "YJIT: Fix naming for a getblockparamproxy counter"

I'm confused. Are they used for different insns?

Reply:
Right, the diff header for this is `gen_getblockparam(. It's for a different generator for a different instruction.